This file is part of MXE.
See index.html for further information.

Source: https://github.com/Alexpux/MINGW-packages/blob/0329cdc2379062087ded826abba65dc870dac373/mingw-w64-libspectre/gs-error-codes.patch

--- a/libspectre/spectre-gs.c
+++ b/libspectre/spectre-gs.c
@@ -43,12 +43,12 @@
 	
 	if (code <= -100) {
 		switch (code) {
-			case e_Fatal:
+			case gs_error_Fatal:
 				fprintf (stderr, "fatal internal error %d", code);
 				return TRUE;
 				break;
 
-			case e_ExecStackUnderflow:
+			case gs_error_ExecStackUnderflow:
 				fprintf (stderr, "stack overflow %d", code);
 				return TRUE;
 				break;
@@ -109,9 +109,9 @@
 		set = _spectre_strdup_printf ("%d %d translate\n", -x, -y);
 		error = gsapi_run_string_continue (ghostscript_instance, set, strlen (set),
 						   0, &exit_code);
-		error = error == e_NeedInput ? 0 : error;
+		error = error == gs_error_NeedInput ? 0 : error;
 		free (set);
-		if (error != e_NeedInput && critic_error_code (error)) {
+		if (error != gs_error_NeedInput && critic_error_code (error)) {
 			fclose (fd);
 			return FALSE;
 		}
@@ -126,7 +126,7 @@
 		read = fread (buf, sizeof (char), to_read, fd);
 		error = gsapi_run_string_continue (ghostscript_instance,
 						   buf, read, 0, &exit_code);
-		error = error == e_NeedInput ? 0 : error;
+		error = error == gs_error_NeedInput ? 0 : error;
 		left -= read;
 	}
 	
--- a/configure.ac
+++ b/configure.ac
@@ -65,6 +65,7 @@
 
 LIBGS_REQUIRED="8.62"
 
+PKG_CHECK_MODULES(GHOSTSCRIPT, [ghostscript >= $LIBGS_REQUIRED], [LIB_GS="$GHOSTSCRIPT_LIBS"], [
 AC_CHECK_LIB(gs, gsapi_new_instance, have_libgs=yes, have_libgs=no)
 if test "x$have_libgs" = "xyes"; then
     LIB_GS="-lgs"
@@ -97,6 +99,7 @@
 else
     AC_MSG_ERROR([You need libgs in order to compile libspectre])
 fi
+])
 AC_SUBST(LIB_GS)
 
 AC_ARG_ENABLE(asserts,
